MOBILE_CLIENT/COMMON/_MEDIUM/XML_INJECTION/description.md

```diff
@@ -0,0 +1,169 @@
+# XML Injection
+
+XML Injection is a type of attack where an attacker injects malicious XML code into an application, which can then be executed by the application. This can lead to a variety of attacks, including denial of service, data theft, and remote code execution.
+
+## External Entity Injection
+
+External Entity Injection is a type of XML Injection where an attacker injects an external entity into an XML document. This can be used to read sensitive data, perform denial of service attacks, or execute remote code.
+
+### Examples
+
+
+#### Swift
+
+```{language}
```

Comment:
These are not formatted correctly. Please recheck your PR content and code before we do the review.